src/state/hooks.ts

```diff
@@ -91,3 +95,32 @@ export const usePriceCakeBusd = (): BigNumber => {
   const farm = useFarmFromPid(pid)
   return farm.tokenPriceVsQuote ? bnbPriceUSD.times(farm.tokenPriceVsQuote) : ZERO
 }
+
+// Alerts
+
+export const useAlert = () => {
+  const dispatch = useDispatch()
+  const helpers = useMemo(() => {
```

Comment:
Saw this used like this in the `use-wallet` repo. I think it is a good idea because the alerts will likely be triggered a lot in effects.